Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Scenes/SampleMain/EVM/EvmCalls.cs

```diff
@@ -0,0 +1,277 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using System.Transactions;
+using ChainSafe.Gaming.MultiCall;
+using ChainSafe.Gaming.UnityPackage;
+using ChainSafe.Gaming.Web3;
+using Nethereum.Contracts.QueryHandlers.MultiCall;
+using Nethereum.Hex.HexConvertors.Extensions;
+using Scripts.EVM.Token;
+using UnityEngine;
+using Web3Unity.Scripts.Prefabs;
+using ABI = ChainSafe.Gaming.UnityPackage.ABI;
+
+public class EvmCalls : MonoBehaviour
+{
+    #region Fields
+    
+    #region IPFS
+
+    private string apiKey = "YOUR_CHAINSAFE_STORE_API_KEY";
+    private string data = "YOUR_DATA";
+    private string bucketId = "BUCKET_ID";
+    private string path = "/PATH";
+    private string filename = "FILENAME.EXT";
+
+    #endregion
+    
+    # region Contract Send
+        
+    private string methodSend = "addTotal";
+    private string abiSend = "[ { \"inputs\": [ { \"internalType\": \"uint8\", \"name\": \"_myArg\", \"type\": \"uint8\" } ], \"name\": \"addTotal\", \"outputs\": [], \"stateMutability\": \"nonpayable\", \"type\": \"function\" }, { \"inputs\": [], \"name\": \"myTotal\", \"outputs\": [ { \"internalType\": \"uint256\", \"name\": \"\", \"type\": \"uint256\" } ], \"stateMutability\": \"view\", \"type\": \"function\" } ]";
+    private string contractAddressSend = "0x7286Cf0F6E80014ea75Dbc25F545A3be90F4904F";
+    private int increaseAmountSend = 1;
+        
+    #endregion
+
+    #region Contract Call
+
+    private string methodCall = "myTotal";
+    private string abiCall = "[ { \"inputs\": [ { \"internalType\": \"uint8\", \"name\": \"_myArg\", \"type\": \"uint8\" } ], \"name\": \"addTotal\", \"outputs\": [], \"stateMutability\": \"nonpayable\", \"type\": \"function\" }, { \"inputs\": [], \"name\": \"myTotal\", \"outputs\": [ { \"internalType\": \"uint256\", \"name\": \"\", \"type\": \"uint256\" } ], \"stateMutability\": \"view\", \"type\": \"function\" } ]";
+    private string contractAddressCall = "0xC71d13c40B4fE7e2c557eBAa12A0400dd4Df76C9";
+
+    #endregion
+
+    #region Get Send Array
+
+    private string contractAddressArray = "0x5244d0453A727EDa96299384370359f4A2B5b20a";
+    private string abiArray = "[{\"inputs\":[{\"internalType\":\"address[]\",\"name\":\"_addresses\",\"type\":\"address[]\"}],\"name\":\"setStore\",\"outputs\":[],\"stateMutability\":\"nonpayable\",\"type\":\"function\"},{\"inputs\":[{\"internalType\":\"uint256\",\"name\":\"\",\"type\":\"uint256\"}],\"name\":\"bought\",\"outputs\":[{\"internalType\":\"address\",\"name\":\"\",\"type\":\"address\"}],\"stateMutability\":\"view\",\"type\":\"function\"},{\"inputs\":[],\"name\":\"getStore\",\"outputs\":[{\"internalType\":\"address[]\",\"name\":\"\",\"type\":\"address[]\"}],\"stateMutability\":\"view\",\"type\":\"function\"}]";
+    private string methodArrayGet = "getStore";
+    private string methodArraySend = "getStore";
```

Comment:
The contents here are the same as line 51

Reply:
fixed